tests/test_util.py

```
@@ -123,28 +123,34 @@ def test_install_tool_success():
-            stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
+            capture_output=True,
+            text=True,
         )
 
 
 @pytest.mark.benchmark
 def test_install_tool_failure():
     """Test _install_tool when pip install fails."""
+
+    def patched_run(*args, **kwargs):
```

Ruff (0.14.3)

Unused function argument: `kwargs`

(ARG001)